ooni/contrib/croniter.py

```diff
@@ -239,7 +239,7 @@ def _get_next(self, ret_type=None, is_prev=False):
             result = self._calc(self.cur, expanded, is_prev)
         self.cur = result
 
-        if ret_type == datetime.datetime:
+        if issubclass(ret_type, datetime.datetime):
```

Comment:
The PR for that has been merged and landed in a new release. We should update our vendorred version of it to be in sync with their latest release: https://github.com/taichino/croniter/blob/master/docs/CHANGES.rst#0313-2016-11-01